### PR TITLE
Nginx 1.18 images reached EOL in November 2022.

### PR DESCRIPTION
**Pull request by phracek**
In November 2022, Nginx 1.18 will reach EOL.

!!!Merge this pull request in December 2022.!!!
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
EOL for imagestreams is here: https://github.com/sclorg/nginx-container/pull/226
[test-all]

**Comment by zmiklank** [comment]
[test]

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
RHEL7 and CentOS 7 failures are fixed by PR #229

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

**Comment by zmiklank** [comment]
Please remove also mentions of version 1.18 from examples/ (examples/Dockerfile, examples/Dockerfile.s2i, examples/README.md).

File: .github/workflows/openshift-tests.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [ "1.18", "1.20", "1.22", "1.22-micro" ]
+        version: [ "1.20", "1.22", "1.22-micro" ]
```
Comment (zmiklank) [inline .github/workflows/openshift-tests.yml:13]: We should remove version specification from the rest of the matrix.

Comment (phracek) [reply .github/workflows/openshift-tests.yml:13]: Fixed by https://github.com/sclorg/nginx-container/pull/225/commits/42fd49a7a67c9e79625ad8259d3594ec20bc8fb6

**Comment by phracek** [comment]
[test-all]

**Review by zmiklank — APPROVED** [review]

**Pull request merged**